helm_deploy/create-and-vary-a-licence-api/templates/app-jobs.yaml

```diff
@@ -0,0 +1,36 @@
+{{ $host := (index .Values "generic-service" "ingress" "host") }}
+{{- range .Values.jobs }}
+  {{- if .enabled -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .name }}
+spec:
+  schedule: {{ .cron }}
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 43200
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 345600 # 4 days
+      backoffLimit: 0
+      parallelism: 1
+      completions: 1
```

Comment:
the email job is not idempotent - there's a lot of effort here in the hope it will only fire the job at most once (this isn't guaranteed by k8s so it's a bit of a best endeavours attempt)